src/driver_custom.c

```diff
@@ -65,8 +65,10 @@ void driver_custom_set(struct app_3bc_s* const app, cpumode_3bc_t cpu_mode,
         }
 
         /** clean wild pointers **/
-        for (; last_func <= atual_func; last_func++) {
+        while (last_func <= actual_func)
+        {
             new_array[last_func] = NULL;
+            ++last_func;
```

Comment:
### error occurred ###
```
In file included from programs/../src/3bc.h:133,
                 from programs/tests.c:2:
programs/../src/driver_custom.c: In function ‘void driver_custom_set(app_3bc_s*, cpumode_3bc_t, register_3bc_t, function_3bc_t)’:
programs/../src/driver_custom.c:6[8](https://github.com/RodrigoDornelles/3bc-lang/actions/runs/3274913903/jobs/5389836982#step:3:9):2[9](https://github.com/RodrigoDornelles/3bc-lang/actions/runs/3274913903/jobs/5389836982#step:3:10): error: ‘actual_func’ was not declared in this scope; did you mean ‘atual_func’?
   68 |         while (last_func <= actual_func)
      |                             ^~~~~~~~~~~
      |                             atual_func
make: *** [Makefile:86: tests-full] Error 1
Error: Process completed with exit code 2.
```

### to solve ###

its my typo wrong, but the **actual** term is still not so correct, replace the `actual_func` and `current_func` in place use `current_func`

Reply:
don't forget to take care of the indentation, you can use clang-format to check.